src/goto-programs/show_goto_functions_json.cpp

```diff
@@ -48,8 +48,11 @@ json_objectt show_goto_functions_jsont::convert(
     json_function["name"]=json_stringt(id2string(function_name));
     json_function["isBodyAvailable"]=
       jsont::json_boolean(function.body_available());
-    bool is_internal=(has_prefix(id2string(function_name), CPROVER_PREFIX) ||
-                      function_name==goto_functions.entry_point());
+    bool is_internal=
```

Comment:
This condition should be pulled out into a separate function that can be accessed from both locations.

Reply:
The two files do not share any dependency where this could be placed conveniently. Anything additional introduced now would be removed by the refactoring that follows.

Reply:
They are both in goto-programs, could you not introduce a `is_internal_function.h` with a single method providing this functionality? But don't worry if the refactoring won't be far behind.